api/subscriptions/subscriptions.go

```diff
@@ -50,10 +51,11 @@ const (
 	pingPeriod = (pongWait * 7) / 10
 )
 
-func New(repo *chain.Repository, allowedOrigins []string, backtraceLimit uint32, txpool *txpool.TxPool) *Subscriptions {
+func New(repo *chain.Repository, allowedOrigins []string, backtraceLimit uint32, txpool *txpool.TxPool, deprecatedEnabled bool) *Subscriptions {
```

Comment:
```suggestion
func New(repo *chain.Repository, allowedOrigins []string, backtraceLimit uint32, txpool *txpool.TxPool, enableDeprecated bool) *Subscriptions {
```

It's better to have the same name convention.

Reply:
Pushed commit